### DISCUSSION TITLE
Unable to set content mode on LottieView

### DISCUSSION THREAD

**Discussion by aslattum** [airbnb/lottie-ios#2497] [General]
## Which Version of Lottie are you using?

Lottie 4.4.3

## Expected Behavior 
Developers should be able to explicitly set the content mode when using LottieView in SwiftUI

## Actual Behavior
Developers have to create a `UIViewRepresentable` around LottieAnimationView` in order to set the content mode.

**Comment by calda** [comment]
Does the `.configure` API not work for this?

Open to adding an official SwiftUI API as well, PRs welcome!

**Reply by aslattum** [reply]
So that actually does work and I didn't know that existed. It would be helpful in the comments of some of the UIKit instructions to reference that `.configure` API!
